src/app/tabs/plugin-tab.js

```diff
@@ -1,12 +1,37 @@
 var yo = require('yo-yo')
-var css = require('./styles/plugin-tab-styles')
+var csjs = require('csjs-inject')
+var remixLib = require('remix-lib')
 
-function plugintab (api = {}, events = {}, opts = {}) {
-  var el = yo`
-    <div class="${css.pluginTabView}" id="pluginView">
-      <iframe class="${css.iframe}" src="${opts.url}/index.html"></iframe>
-    </div>`
-  return { render () { return el } }
-}
+var EventManager = remixLib.EventManager
 
-module.exports = plugintab
+module.exports = class plugintab {
+  constructor (api = {}, events = {}, opts = {}) {
+    const self = this
+    self.event = new EventManager()
+    self._api = api
+    self._events = events
+    self._view = { el: null }
+    self.data = { url: opts.url }
```

Comment:
why not directly `self._opts = opts`, like for the other param or maybe `self._url = opts.url` ? 
The code is totally fine, i would not object it ...  but i am not sure I would add this `.data` rule as a best practices if you ever want it.

Reply:
yes i agree - the `.data` was basically some kind of idea for a convention before we started to do `._opts` ...so i agree and will change that :-)